runtime/include/tt/runtime/types.h

```diff
@@ -120,9 +122,20 @@ struct Event : public detail::RuntimeCheckedObjectImpl {
 
 struct Tensor : public detail::RuntimeCheckedObjectImpl {
   std::shared_ptr<void> data;
+  int volume;
```

Comment:
I think we can revert back to not having this member now right?

Reply:
fixed